tasmota/tasmota_xsns_sensor/xsns_40_pn532.ino

```diff
@@ -393,93 +435,232 @@ uint8_t mifareclassic_WriteDataBlock (uint8_t blockNumber, uint8_t *data) {
   return (0 < PN532_readResponse(Pn532.packetbuffer, sizeof(Pn532.packetbuffer)));
 }
 
+uint8_t ntag21x_probe (void) {
```

Comment:
Prefix ALL functions with PN532_ as they are global.

Reply:
Done. I have added a prefix to ALL the functions

Reply:
@arendst, please tell me on the issue of saving the password and the pack in the global configuration (to save it on reboot). Can you do it or should I do it?

Reply:
I'll prepare a location in Settings. Once done I let you do the implementation in PN532

Reply:
Just added ``Settings->pn532_password`` and ``Settings->pn532_pack`` for you to play with.